rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -846,4 +864,16 @@ private BtcECKey findPublicKeySignedBy(List<BtcECKey> pubs, BtcECKey pk) {
         }
         return pk;
     }
+
+    private FederationSupport createFederationSupport(FederationConstants federationConstants, FederationStorageProvider federationStorageProvider, Block executionBlock, ActivationConfig.ForBlock activations) {
```

Comment:
You can create a test builder for FederationSupport, which, by default, mocks everything. Then use the builder where needed instead of this kind of method. Like [BridgeSuppotBuilder](https://github.com/rsksmart/rskj/blob/f965b7d79cf5538787fc7c41bfe582ae12effa65/rskj-core/src/test/java/co/rsk/test/builders/BridgeSupportBuilder.java#L32)

Reply:
Nice, but let's do that in a separate ticket